### ISSUE TITLE
Shared Poco libs windows, link error (missing ws2_32)

### ISSUE THREAD

**Issue by lasote** [pocoproject/poco#1766]
Building **shared** Poco libraries (NetSSL_Win and Crypto) in Windows, it's needed to link with **ws2_32**.

Also Crypto shared requires **Crypt32.lib** (NetSSL_Win already have it).

 **(NetSSL_Win)**
```
Net Util Foundation Crypt32.lib => Net Util Foundation Crypt32.lib ws2_32
```
 **(Crypto)**
```
Foundation ${OPENSSL_LIBRARIES} => Foundation ${OPENSSL_LIBRARIES} ws2_32 Crypt32.lib
```

**Comment by aleks-f** [comment]
which version of VS? builds fine with VS2015

**Comment by lasote** [comment]
Yes, VS 2015. The error only happens building the Shared libraries.

**Comment by zosrothko** [comment]
which version of poco? which build command are you running?

**Comment by OgreTransporter** [comment]
Fixed: #1733
